sufsara_tutorial.Rmd

```diff
@@ -39,48 +65,28 @@ Press OK, and from here on you can press the bubble on the right-hand side to la
 
 ![VM running through X2go](figs/x2gorunning.PNG)
 
+<!-- Likely all deprecated, and possibly also not relevant
```

Comment:
Correct, the VMs actually don't even let you shut down (I removed the button for that). There is still a button to restart, but by default it also requires authentication (which is the password of the ubuntu user, which does not have a password set, unless you run `sudo passwd ubuntu` beforehand).